exposure-notification/src/main/java/fi/thl/covid19/exposurenotification/diagnosiskey/DiagnosisKeyService.java

```diff
@@ -66,7 +66,7 @@ private long getExportedKeyCount(List<TemporaryExposureKey> keys) {
                 getRiskBucket(symptomsOnset, utcDateOf10MinInterval(requestKey.rollingStartIntervalNumber)),
                 requestKey.rollingStartIntervalNumber,
                 requestKey.rollingPeriod,
-                requestKey.visitedCountries,
+                requestKey.visitedCountries.keySet(),
```

Comment:
I expected to see the filtering of value != 1 here. Where is it?

Reply:
Oh, right below. The field inside the object might as well be the set directly; if you're filtering in the parsing part (which is fine), the map structure is pointless after that.

Reply:
It's complicates e.g. tests, so that's why this is it in this way.